api/managementpb/dbaas/json/dbaas.json

```diff
@@ -2754,6 +2785,37 @@
                       "format": "int32",
                       "x-order": 0
                     },
+                    "haproxy": {
+                      "description": "HAProxy container parameters.\nNOTE: HAProxy does not need disk size as ProxySQL because the container does not require it.",
```

Comment:
```suggestion
                      "description": "HAProxy container parameters.\nNOTE: HAProxy does not need disk size as ProxySQL does because the container does not require it.",
```